Project.toml

```diff
@@ -1,14 +1,14 @@
 name = "StructTypes"
 uuid = "856f2bd8-1eba-4b0a-8007-ebc267875bd4"
 authors = ["Jacob Quinn"]
-version = "1.1"
+version = "1.2.0"
 
 [deps]
-UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
-julia = "1"
+julia = "1.2"
```

Comment:
Why this change?

Reply:
Originally this was because of `hasfield` requiring Julia 1.2. But now we have a way of making it work on Julia 1.0 and 1.1, so this change no longer exists.